manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.zh-CN.yaml

```diff
@@ -0,0 +1,16 @@
+# Automatically updated by the winget bot at 2021/Jul/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: 15.0.18386.0
+PackageLocale: zh-CN
+Publisher: Microsoft Corporation
+PublisherUrl: https://docs.microsoft.com/zh-CN/sql/ssms/download-sql-server-management-studio-ssms
+Author: Microsoft
+PackageName: Microsoft SQL Server Management Studio
+PackageUrl: https://docs.microsoft.com/zh-CN/sql/ssms/download-sql-server-management-studio-ssms
+License: 版权所有 (c) Microsoft Corporation
+Copyright: 版权所有 (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
```

Comment:
I have no idea what the Description says, can you add a better ShortDescription?

Reply:
Sure. I just updated the short description. :)